user_management/api/serializers.py

```diff
@@ -31,6 +38,7 @@ class Meta:
 
 
 class RegistrationSerializer(ValidateEmailMixin, serializers.ModelSerializer):
+    email = serializers.EmailField(validators=[unique_email_validator])
```

Comment:
This doesn't work - I think it's something to do with the metaclass `Serializer` uses.